src/examples/notify_with_inferred_template.yml

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
I recommend not including this example. The name isn't obvious.  and the example is long but can not really demonstrate the outcome.

Reply:
Should I include an example that makes more sense or no example at all?